src/CMakeLists.txt

```diff
@@ -639,9 +639,11 @@ install(PROGRAMS
 add_subdirectory(bash_completion)
 add_subdirectory(client)
 
+if((WITH_LIBCEPHFS OR WITH_KRBD) AND NOT FREEBSD)
```

Comment:
i'd recommend put
```cmake
find_package(keyutils REQUIRED)
```
where keyutils is actually used. even if we need to have two `find_package(keyutils REQUIRED)` at different places.

this `if` block is just confusing.

Reply:
I have take this back to `./CMakeLists.txt` where we detect more packages.
And based on the availability `HAVE_KEYUTILS` the remainder of the code selected.